bedrock/firefox/templates/firefox/personal.html

```diff
@@ -37,6 +37,9 @@
     <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
       {{ high_res_img('firefox/android/btn-google-play.png', {'alt': 'Get it on Google Play', 'width': '172', 'l10n': True}) }}
     </a>
+    <small class="download-other">
+      <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+    </small>
```

Comment:
Here also, contrast and legibility can be improved:

![screen shot 2015-09-14 at 12 07 13](https://cloud.githubusercontent.com/assets/10350960/9847535/e44c91cc-5ad9-11e5-826a-f9c4c5b810fa.png)

Here I would suggest:

```
.download-other {
    a {
        color: #fff;
        .open-sans-light();
        font-size: .95rem;
    }
}
```
![screen shot 2015-09-14 at 12 11 59](https://cloud.githubusercontent.com/assets/10350960/9847584/2bcb270c-5ada-11e5-9a62-a84460307d93.png)